install_files/ansible-base/roles/build-securedrop-app-code-deb-pkg/files/usr.sbin.apache2

```diff
@@ -129,6 +129,23 @@
   /var/www/securedrop/dictionaries/nouns.txt r,
   /var/www/securedrop/journalist.py r,
   /var/www/securedrop/journalist.pyc rw,
+  /var/www/securedrop/journalist_app/__init__.py r,
+  /var/www/securedrop/journalist_app/__init__.pyc rw,
```

Comment:
this is a nit (kind of talking outloud here - you certainly dont need to trial it for this PR) -- but I wonder if we can do PCRE notation on these ... 
looks like:  `var/www/securedrop/journalist_app/__init__.{pyc,py} rw` might work....

http://wiki.apparmor.net/index.php/QuickProfileLanguage

Reply:
oh im dumb -- missed the alternating `r` / `rw`s

Reply:
I think it's a good thing that the app can't write to it's own source files, actually.

Reply:
Principle of least privilege 'n all.

Reply:
🐩